source/github-helpers/selectors.ts

```diff
@@ -10,7 +10,7 @@ export const repoUnderlineNavUl_ = [
 ] satisfies UrlMatch[];
 
 export const standaloneGistLinkInMarkdown = css`
-	.js-comment-body p a:only-child:is(
+	:is(.js-comment-body, .react-issue-comment) p a:only-child:is(
```

Comment:
Tip: If you see simple class changes like these, it might be good to search them on the repo and fix multiple features at once. For example there's `rgh-linkify-features` which will need this change eventually.

Reply:
👍 Other pr.